docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3"
```

Comment:
docker-compose cam be changed with:

```yml
version: "3"
services:
  php:
    build:
      context: .
      args:
        - PHP_VERSION=${PHP_VERSION:-7.2}
      dockerfile: docker/Dockerfile
    depends_on:
      - mongo
    volumes:
      - ./:/docker

  mongo:
    image: mongo:latest
```

You can optionally pass `PHP_VERSION` as environment variable in `.travis.yml`, using it for different PHP versions